builtins/amp-video.js

```diff
@@ -98,6 +121,80 @@ export function installVideo(win) {
     isVideoSupported_() {
       return !!this.video_.play;
     }
+
+    /**
+     * VideoInterface Implementation
+     * {@see ../src/video-interface.VideoInterface}
+     */
+
+    /**
+     * @return {boolean}
+     */
+    supportsPlatform() {
+      return this.isVideoSupported_();
+    }
+
+    /**
+     * @return {boolean}
+     */
+    hasAutoplay() {
+      return this.element.hasAttribute('autoplay');
```

Comment:
yep, now `autoplay` and `controls` are expected attributed and no longer part of the JS interface (but documented same way the events are in the `video-interface.js`)